js/join.js

```diff
@@ -126,32 +384,36 @@ function showPage(currentFlowState) {
 //Validators
 function arePersonalDetailsValid() {
   return (
-    state.value.trim().length > 3 &&
-    city.value.trim().length > 3 &&
-    country.value.trim().length > 3
+    state.value.trim().length >= 3 &&
+    city.value.trim().length >= 3 &&
+    country.value.trim().length >= 3
   );
 }
 
 function introPageValidator() {
   return (
-    introduction.value.trim().split(' ').length > 100 &&
-    skills.value.trim().split(' ').length > 6 &&
-    college.value.trim().split(' ').length > 5 &&
-    forFun.value.trim().split(' ').length > 100 &&
-    funFact.value.trim().split(' ').length > 100
+    introduction.value.trim().split(' ').length >= 100 &&
+    skills.value.trim().split(' ').length >= 5 &&
+    college.value.trim().split(' ').length >= 5 &&
+    forFun.value.trim().split(' ').length >= 100 &&
+    funFact.value.trim().split(' ').length >= 100
   );
 }
 
 function whyRdsPageValidator() {
-  return whyRds.value.trim().split(' ').length > 100 && foundFrom.value != '';
+  return whyRds.value.trim().split(' ').length >= 100 && foundFrom.value != '';
 }
 
 function dataValidator(element, size) {
```

Comment:
Is this function used at multiple places?
Because this function is called from `getFromLocalStorage`, and it's commented there.

Reply:
i did not get you

Reply:
As per the discussion i understood